Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM ubuntu:focal-20220922
```

Comment:
Could you be so kind and use the base image ubuntu:focal-20221019? (Just to reduce the docker image size.)